docs/configuration.md

```diff
@@ -2622,11 +2622,27 @@ Please refer to the [Security](security.html) page for available options on how
 Spark subsystems.
 
 
-{% for static_file in site.static_files %}
-    {% if static_file.name == 'generated-sql-configuration-table.html' %}
 ### Spark SQL
 
-{% include_relative generated-sql-configuration-table.html %}
+#### Runtime SQL Configuration
+
+Runtime SQL configurations inner-session, mutable Spark SQL configurations. They can be set and queried by SET commands and rest by RESET command, or by `SparkSession.conf`'s setter and getter methods.
+
+{% for static_file in site.static_files %}
+    {% if static_file.name == 'generated-runtime-sql-config-table.html' %}
+        {% include_relative generated-runtime-sql-config-table.html %}
+        {% break %}
+    {% endif %}
+{% endfor %}
+
+
+#### Static SQL Configuration
+
+Static SQL configurations are cross-session, immutable Spark SQL configurations. External users can query the static sql config values via `SparkSession.conf` or via set command, e.g. `SET spark.sql.extensions;`, but cannot set/unset them.
```

Comment:
We should also mention how to set them, e.g. by config file or by setting `SparkConf` that is used to create `SparkSession`.